styles.css

```diff
@@ -15,5 +20,63 @@ border-color: hotpink;
 
```

Comment:
How about creating a fixed header at the top of your page?
You could use e.g. `property:sticky;` and `top: 0;`.

Reply:
Oh thank you so much, Alex! Great ideas, I will do that ! :)